gnovm/cmd/gno/main.go

```diff
@@ -41,8 +42,7 @@ func newGnocliCmd(io commands.IO) *commands.Command {
 		// publish/release
 		// generate
 		// "vm" -- starts an in-memory chain that can be interacted with?
-		// bug -- start a bug report
-		// version -- show gno, golang versions
+		// version -- show gnodev, golang versions
```

Comment:
```suggestion
		// version -- show cmd/gno, golang versions
```

(let's put cmd/ to specify that we're talking about the binary)